docker-image/run-nlp-profiler-in-docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+MOUNT_VOLUME="${PWD}/../"
+TARGET_FOLDER="/nlp_profiler"
+DOCKER_IMAGE_NAME="nlp_profiler"
+
+echo "~~~ Running nlp_profiler in a docker container"
+echo "Mounted volume: ${MOUNT_VOLUME}"
+
```

Comment:
Its a good practise to `echo`  the steps that are executed at the different lines in the code  - this way when one reads the console they can tell what stage of the shellscript it is at - look at examples of my own code to find out how we have been doing it, the clue is look for `echo`s in the shell script and also in the `Dockerfile`